packages/beachball/src/__e2e__/publishRegistry.test.ts

```diff
@@ -21,6 +21,57 @@ describe('publish command (registry)', () => {
     await registry.reset();
   });
 
+  it('will perform retries', async () => {
```

Comment:
add check for retry is actually happening?

Reply:
sure